composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "2d1dfc8bddd169a9064a89c693bb5218",
+    "content-hash": "ec5c7533d5c9d49d54530eb94958e55b",
```

Comment:
Is this content hash correct? I'm getting a warning about composer.lock being out of date and you can see the same message on PR test console logs. When I try to update it it also seems to remove a whole bunch of other stuff but I'm not sure if it should or not.

I also get a message `License "AGPL-3.0" is a deprecated SPDX license identifier, use "AGPL-3.0-only" or "AGPL-3.0-or-later" instead`